cmd/podman/login.go

```diff
@@ -90,7 +91,8 @@ func loginCmd(c *cliconfig.LoginValues) error {
 
 	// username of user logged in to server (if one exists)
 	userFromAuthFile, passFromAuthFile, err := config.GetAuthentication(sc, server)
-	if err != nil {
+	// Do not return error if no credentials found in credHelpers, new credentials will be stored by config.SetAuthentication
+	if err != nil && err != credentials.NewErrCredentialsNotFound() {
```

Comment:
Should be errors.Cause(err) instead of err, to be safe (we might wrap the error at some later point)?

Reply:
this err is not wrapped in `config.GetAuthentication` -> `findAuthentication(registry, path, legacyFormat)`. 
you mean wrap it in `config.GetAuthentication` and use errors.Cause(err) here?